samcli/local/docker/container.py

```diff
@@ -272,6 +274,31 @@ def start(self, input_data=None):
         # Start the container
         real_container.start()
 
+        # Wait for port to be open
+        self.wait_for_port()
+
+    def wait_for_port(self):
+        """
+        Waits until the host machine port that Docker binds to is open.
+        """
+        start_time = time.time()
+        sleep = 0.1
+        timeout = 10
```

Comment:
can you define these values as Constants values per the module. I believe this will help you customizing them during unit testing

Reply:
Also having `timeout` as a method parameter might make it easier to configure later on and would make testing a  little easier.

Reply:
decided to make it a constant for now, as it can also be patched in the tests. I think if later on we found that we wanted to wait for a port in other parts of the code, then maybe we could put this in some util in a more generic way (accepting timeout, sleep, port, etc. parameters).